modules/http/http-worker.c

```diff
@@ -289,61 +289,91 @@ _finish_request_body(HTTPDestinationWorker *self)
     g_string_append_len(self->request_body, owner->body_suffix->str, owner->body_suffix->len);
 }
 
-static LogThreadedResult
-_flush_on_target(HTTPDestinationWorker *self, HTTPLoadBalancerTarget *target)
+static void
+_debug_flush_on_target(HTTPDestinationWorker *self, HTTPLoadBalancerTarget *target, glong http_code)
```

Comment:
May I suggest calling `_debug_response_info`. The current name is a little confusing as no flush is happening here.